src/Game.py

```diff
@@ -11,19 +11,34 @@ def __init__(self):
         self.background = self.background.convert()
         self.background.fill((250, 250, 250))
         self.isRunning = False
+        
 
     def runGame(self):
         pygame.init()
+        clock = pygame.time.Clock()
+        player = Player(50,450,20,20)
+        step = 5
```

Comment:
There is no point in defining variables here.. Step should be a property of game

Reply:
chANged